manifests/init.pp

```diff
@@ -120,15 +74,15 @@
     }
   }
 
-  if ($create_symlink == true and $symlink_target != undef) {
+  if $create_symlink == true and $symlink_target != undef {
     file { 'arc_symlink':
       ensure => link,
       path   => '/usr/lib/libtcl.so.0',
       target => $symlink_target,
     }
   }
 
-  if $install_package == true and $packages != undef {
+  if $install_package == true {
     package { $packages:
```

Comment:
Should we maybe switch this to ensure_packages instead? This will ensure it doesn't cause issues with other modules/classes including the same packages?

Reply:
will do so